### ISSUE TITLE
Remove "Validation" suffix from all annotations and bindings.

### ISSUE THREAD

**Issue by WellingtonCosta** [WellingtonCosta/convalida#42]
I would like to hear the your opinion @emanuelet, @kaio-henrique, @itscorey.

**Comment by itscorey** [comment]
IMO, they should have `Field` as *prefix* in some cases.

```@RequiredField(errorMessage = "Error", autoDismiss = true)```

In some others cases it will not be needed.

```
@Email()
@ConfirmEmail()
@CPF()
@Length()
[...]
```

**Comment by WellingtonCosta** [comment]
I think adding a prefix in only some cases can be confused.

My idea about remove `Validation` suffix is to turn API less verbose.

Something like this: https://docs.jboss.org/hibernate/beanvalidation/spec/2.0/api/

**Comment by itscorey** [comment]
> I think adding a prefix in only some cases can be confused.
> 
> My idea about remove `Validation` suffix is to turn API less verbose.
> 
> Something like this: https://docs.jboss.org/hibernate/beanvalidation/spec/2.0/api/

- `@Required` may conflict with other libraries.
- `@Length` doesn't seems friendly (length for what?). Idk.

Anyway, removing the `Validation` suffix is valid.

**Comment by WellingtonCosta** [comment]
> * `@Required` may conflict with other libraries.

 `@Required` doesn't conflit with other libraries because the packages aren't the same.

> * `@Length` doesn't seems friendly (length for what?). Idk.

Well, it's another discussion. What do you think when you apply `@Length(min=3, max=10, errorMessage=R.string.invalid_length)` to a field?

**Comment by itscorey** [comment]
> > * `@Required` may conflict with other libraries.
> 
> `@Required` doesn't conflit with other libraries because the packages aren't the same.
> 
> > * `@Length` doesn't seems friendly (length for what?). Idk.
> 
> Well, it's another discussion. What do you think when you apply `@Length(min=3, max=10, errorMessage=R.string.invalid_length)` to a field?

That's seems ok.

**Comment by kaio-henrique** [comment]
I think it's okay to remove the suffix. The library itself already makes clear what its purpose is. Keeping the "Validation" suffix to me becomes redundant. 😄 🔥

**Comment by emanuelet** [comment]
Yeah, I'm on the same page. Removing the suffix is not a bad idea
